modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -47,3 +46,33 @@ reviews-v3-6dfd49b55b-vcwpf       2/2     Running   0          12m
 echo "http://$GATEWAY_URL/productpage"
 ----
 . Copy and paste the output in a web browser to verify the Bookinfo product page is deployed.
+
+.Procedure from Kiali web console
+
+. Obtain the address for the Kiali web console.
```

Comment:
This is probably fine but why do you need this if you are linking directly from openshift console?  Is {product-title} below going to be openshift or ossm?  It would need to be openshift, I think.

Reply:
Julie and I thought it would be helpful to include this for new users. Some of the general feedback doc has heard is that customers find it difficult to use Red Hat products. We're just trying to help customers.

The variable on line 54 gets populated with OpenShift. You can see how the variables resolves in the preview build: https://deploy-preview-44411--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh.html#ossm-tutorial-bookinfo-verify-install_ossm-create-mesh